apps/main/src/dex/components/PageCreatePool/TokensInPool/SelectTokenButton.tsx

```diff
@@ -168,6 +168,7 @@ const SelectTokenButton = ({
           disabledTokens={disabledKeys ?? []}
           disableSorting={true}
           disableMyTokens={false}
+          compact={false}
```

Comment:
compact is optional bool, the prop could be ommitted
```suggestion
```

Reply:
It's only optional on the TokenSelector root component, not the modal component. Since this is an old style button only the modal component is needed here, hence the big amount of props. This was a deliberate choice as I'd rather not having a chain of intermediate optionals resorting to different defaults.